documentation/index.md

```diff
@@ -27,15 +27,43 @@ Topology probes currently implemented :
 * LLDP
 * LibVirt
 
-## Topology connectors
+## Analayzer Topology Probes
+
+Following are analyzer graph topology probes:
 
 * Neutron
 * Docker
-* Kubernetes
+* K8s (Kubernetes)
```

Comment:
Why not simply "Kubernetes" ?

Reply:
Maybe we could display the "internal" name for each probe, like :
* Neutron (`neutron`)
* Docker (`docker`)
* Kubernetes (`k8s`)
* Lxd (`lxd`)
* Opencontrail (`opencontrail`)
* runC (`runc`)

Reply:
By the way, Neutron, Docker Lxd, Opencontrail and runC are not analyzer probes, but agent probes.

Reply:
ok - will handle all the above